schemas/IDLs/org.gel.models.participant.avro/1.2.0/Ngis.avdl

```diff
@@ -117,5 +119,9 @@ protocol Ngis {
         Cancer Participant
         */
         union {null, CancerParticipant} cancerParticipant;
+        /**
+        Clinical Indication of the referral
+        */
+        ClinicalIndication clinicalIndication;
```

Comment:
I believe this should be a list. Currently we do have cases with more than one clinical indication.

Reply:
Clinical Indications are new for NGIS and there will only be one.  We can make it a list and future proof it, but the Clinicians have been super adamant that there is only one Clinical Indication per referral. Your thoughts @Antonior26 @priesgo

Reply:
If we want to fit the current interpretation request in the referral at any moment this won't fit. But maybe we don't want to do that at all.

Reply:
agree with @cjohnson12, this reflect the current design we should not over-complicate it

Reply:
ok